sda-mq/templates/statefulset.yaml

```diff
@@ -97,6 +101,18 @@ spec:
       containers:
       - name: broker
         image: "{{ include "image.registry" . }}{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+        {{- if .Values.vaultSecrets }}
+        command:
+          - /bin/sh
+          - -c
+          - |
+            source /vault/secrets/pws
+          {{- if and .Values.config.shovel.host (and .Values.config.shovel.user .Values.config.shovel.pass) }}
```

Comment:
if these lines happen shouldn't we exclude the other env var: https://github.com/CSCfi/sda-helm/blob/554c44ece639878a2696f2b1c6344502ea26cf00/sda-mq/templates/statefulset.yaml#L156-L162 ?